distro/configs/openmrs/frontend_assembly/spa-assemble-config.json

```diff
@@ -1,4 +1,5 @@
-{
+{  
+  "coreVersion": "latest",
   "frontendModules": {
     "@openmrs/esm-cohort-builder-app": "latest",
     "@openmrs/esm-home-app": "latest",
```

Comment:
I think we could add these two apps into this file, since we had them in the refApp spa assemble file (Deleted above)
`"@openmrs/esm-billing-app": "next"`
 `"@openmrs/esm-stock-management-app": "next"`